.appveyor.yml

```diff
@@ -1,186 +1 @@
-      BUILD_COMMIT: ff3df08438d570b0ccdda3f8a008278d8a4ad394
-      DAILY_COMMIT: master
-
-  matrix:
-    - PYTHON: C:\Python36
-      PYTHON_VERSION: 3.6
-      PYTHON_ARCH: 32
-
-    - PYTHON: C:\Python36-x64
-      PYTHON_VERSION: 3.6
-      PYTHON_ARCH: 64
-
-    - PYTHON: C:\Python37
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 32
-
-    - PYTHON: C:\Python37-x64
-      PYTHON_VERSION: 3.7
-      PYTHON_ARCH: 64
-
-    - PYTHON: C:\Python38
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 32
-
-    - PYTHON: C:\Python38-x64
-      PYTHON_VERSION: 3.8
-      PYTHON_ARCH: 64
-
-
-init:
-  - "ECHO %PYTHON% %PYTHON_VERSION% %PYTHON_ARCH%"
-  - "ECHO \"%APPVEYOR_SCHEDULED_BUILD%\""
-  - ps: >-
-      if ($env:APPVEYOR_REPO_BRANCH -eq "master") {
-        $env:CONTAINER = "pre-release"
-        if ($env:DAILY_COMMIT) { $env:BUILD_COMMIT = $env:DAILY_COMMIT }
-        $env:NPY_RELAXED_STRIDES_DEBUG = 1
-      } else {
-        $env:CONTAINER = "wheels"
-        $env:UPLOAD_ARGS = "--no-update-index"
-      }
-install:
-  - cmd: echo "Filesystem root:"
-  - dir C:\
-
-  - echo "Installed SDKs:"
-  - dir "C:/Program Files/Microsoft SDKs/Windows"
-
-  # Get needed submodules
-  - git submodule update --init
-
-  # Install new Python if necessary
-  - ps: .\multibuild\install_python.ps1
-
-  # Prepend required Python to the PATH of this build (this cannot be
-  # done from inside the powershell script as it would require to restart
-  # the parent CMD process).
-  - SET PATH=%PYTHON%;%PYTHON%\Scripts;%PATH%
-
-  # Check that we have the expected version and architecture for Python
-  - python --version
-  - >-
-     python -c "import sys,platform,struct;
-     print(sys.platform, platform.machine(), struct.calcsize('P') * 8, )"
-
-  # Download and install static "openblas.a" to PYTHON\lib
-  - ps: |
-      $PYTHON_ARCH = $env:PYTHON_ARCH
-      $PYTHON = $env:PYTHON
-      If ($PYTHON_ARCH -eq 32) {
-          $OPENBLAS = $env:OPENBLAS_32
-      } Else {
-          $OPENBLAS = $env:OPENBLAS_64
-      }
-      $clnt = new-object System.Net.WebClient
-      $file = "$(New-TemporaryFile).zip"
-      $tmpdir = New-TemporaryFile | %{ rm $_; mkdir $_ }
-      $destination = "$PYTHON\lib\openblas.a"
-      echo $file
-      echo $tmpdir
-      echo $OPENBLAS
-      $clnt.DownloadFile($OPENBLAS,$file)
-      Expand-Archive $file $tmpdir
-      rm $tmpdir\$PYTHON_ARCH\lib\*.dll.a
-      $lib = ls $tmpdir\$PYTHON_ARCH\lib\*.a | ForEach { ls $_ } | Select-Object -first 1
-      echo $lib
-      cp $lib $destination
-      ls $destination
-
-  # Upgrade to the latest pip, setuptools, and wheel.
-  - python -m pip install -U pip setuptools wheel
-
-  # Install build requirements.
-  - pip install "%CYTHON_BUILD_DEP%" --install-option="--no-cython-compile"
-
-build_script:
-  - cd numpy
-  - git checkout %BUILD_COMMIT%
-  # Create _distributor_init.py
-  - cd ..
-  - python -c "import openblas_support; openblas_support.make_init('numpy/numpy')"
```

Comment:
ah, there is someone using `openblas_support`. This should be using the `openblas_support` from `numpy/tools` and not a local copy. I think I did it this way since we were building wheels before `numpy/tools/openblas_support.py` existed, but now we can fix this correctly